docs/src/main/sphinx/functions.rst

```diff
@@ -8,7 +8,9 @@ Trino operating on the underlying data sources.
 
 If you are looking for a specific function or operator, see the :doc:`full
 alphabetical list</functions/list>` or the :doc:`full list by
-topic</functions/list-by-topic>`.
+topic</functions/list-by-topic>`. Using :doc:`SHOW FUNCTION
+</sql/show-functions>` may yield information on additional arguments and
+behaviors.
```

Comment:
It's `SHOW FUNCTIONS`, plural, but that's an unsatisfying answer to the originating complaint. There are 743 functions, which is what `SHOW FUNCTIONS` displays by default, paging them a screenful at a time. That's no way to find the details for an individual function. 

Could we at least add an example of using LIKE to narrow down the list you see? For example:

`SHOW FUNCTIONS LIKE '%url%';`
displays only the 12 functions with `url` in their names.

or 

`SHOW FUNCTIONS LIKE '%zip%';`
displays only the 6 functions with `zip` in their names so that readers don't have to scroll to the end of 743 functions to find them.

Reply:
That is already documented in the linked SHOW FUNCTIONS page. Don't want to repeat the same information in two places. 

Will fix the plural issue.

Reply:
See the linked page https://trino.io/docs/current/sql/show-functions.html

Reply:
And yes.. dont repeat the info we already have in show functions .. if anything .. build it out more there..

Reply:
The above examples are fundamentally the same as the example that is already on the show-functions.rst page and don't really impart any new information, which is (excerpt):

:ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to
filter the results to the desired subset. For example, the following query
allows you to find functions beginning with ``array``::

    SHOW FUNCTIONS LIKE 'array%';

The Specify a pattern link already covers how to use `LIKE`.